guides/source/active_record_validations.md

```diff
@@ -1650,16 +1757,16 @@ irb> person.errors.empty?
 Displaying Validation Errors in Views
 -------------------------------------
 
-Once you've created a model and added validations, if that model is created via
-a web form, you probably want to display an error message when one of the
-validations fails.
+Once you've defined a model and added validations, you'll want to display an
+error message when a validation fails during the creation of that model via a
+web form.
```

Comment:
could we simplify the sentence like this? Since the validation can fail at times other than "creation of the model" maybe no need to mention that?
```suggestion
Once you've defined a model and added validations, you'll want to display an
error message to the user via a web form when a validation fails.
```